microovn/lib/charms/ovsdb_interface/v0/ovsdb.py

```diff
@@ -0,0 +1,125 @@
+"""TODO: Add a proper docstring here.
+
+This is a placeholder docstring for this charm library. Docstrings are
+presented on Charmhub and updated whenever you push a new version of the
+library.
+
+Complete documentation about creating and documenting libraries can be found
+in the SDK docs at https://juju.is/docs/sdk/libraries.
+
+See `charmcraft publish-lib` and `charmcraft fetch-lib` for details of how to
+share and consume charm libraries. They serve to enhance collaboration
+between charmers. Use a charmer's libraries for classes that handle
+integration with their charm.
+
+Bear in mind that new revisions of the different major API versions (v0, v1,
+v2 etc) are maintained independently.  You can continue to update v0 and v1
+after you have pushed v3.
+
+Markdown is supported, following the CommonMark specification.
+"""
+
+# The unique Charmhub library identifier, never change it
+LIBID = "599e7729d8cf403db3f6afb6d7c64c92"
+
+# Increment this major API version when introducing breaking changes
+LIBAPI = 0
+
+# Increment this PATCH version before using `charmcraft publish-lib` or reset
+# to 0 if you are raising the major API version
+LIBPATCH = 1
+
+ENV_FILE = "/var/snap/microovn/common/data/env/ovn.env"
+CONNECT_ENV_NAME = "OVN_{0}_CONNECT"
+CONNECT_STR_KEY = "db_{0}_connection_str"
```

Comment:
I like that you have made this singular :+1:

(the plural form [in the existing consuming charms](https://opendev.org/openstack/sunbeam-charms/src/commit/0d41be650d94dd5dffd8107db90a68ee62d695bf/charms/neutron-k8s/src/templates/ml2_conf.ini.j2#L35) does not make sense to me)